src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/PropertyInfoPassTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\Compiler;
+
+use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\PropertyInfoPass;
+use Symfony\Component\DependencyInjection\Reference;
+
+class PropertyInfoPassTest extends \PHPUnit_Framework_TestCase
+{
+    public function testServicesAreOrderedAccordingToPriority()
+    {
+        $services = array(
+            'n3' => array('tag' => array()),
+            'n1' => array('tag' => array('priority' => 200)),
+            'n2' => array('tag' => array('priority' => 100)),
+        );
+
+        $expected = array(
+            new Reference('n1'),
+            new Reference('n2'),
+            new Reference('n3'),
+        );
+
+        $container = $this->getMock('Symfony\Component\DependencyInjection\ContainerBuilder', array('findTaggedServiceIds'));
+
+        $container->expects($this->atLeastOnce())
```

Comment:
I suggest we avoid checking number of calls with stubs and use `$this->any()` instead. It's not important how many times a stubbed method is called. It's important that if it's called, it returns the stubbed value. Relying on number of calls to stubbed methods makes refactorings harder. It ties tests to the implementation.

There's a similar case in the next test.